CONTRIBUTING.md

```diff
@@ -9,8 +9,9 @@ Before getting started, go through the following:
 1. Read and observe the [code of conduct](CODE_OF_CONDUCT.md).
 2. Sign the [CLA](#cla).
 3. Check out the [Architecture document](/docs/architecture.md) for the Antrea architecture and design.
-4. Set up necessary [accounts](#accounts-setup).
-5. Set up your [development environment](docs/manual-installation.md)
+4. Familiarize yourself with the [OVS flows and the pipeline](/docs/ovs-pipeline.md).
```

Comment:
Feel the steps listed here are all high level, maybe it's more proper to link OVS flows doc in `architecture.md`?

Reply:
+1 for a link in the architecture document

Reply:
done